frontend/packages/ui/src/components/Toast/Toast.module.css

```diff
@@ -15,25 +15,29 @@
 }
 
 .wrapper.info {
-  background: var(--info-background);
```

Comment:
These `--xxx-background` have been named to `--toast-xxx-background`. https://github.com/liam-hq/liam/commit/423066159cebfa8c1ac0e7ea6e07c770529e6375